.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,8 @@
 Closes #<ticket number>
 
+<!-- Please do not remove this, even if you think you don't need it -->
 ### Successful PR Checklist:
 - [ ] Tests
 - [ ] Documentation (optionally: [run spell checking](https://github.com/peopledoc/procrastinate/blob/master/CONTRIBUTING.rst#build-the-documentation))
+- [ ] Migration, if applicable
```

Comment:
```suggestion
- [ ] Migration procedure, if applicable
```

Reply:
I've changed it to "SQL Migrations". This refers to [this part](https://github.com/peopledoc/procrastinate/blob/master/CONTRIBUTING.rst#migrations) in the contributing doc.